server/start_server.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
-
-tmux new-session -s rematch -d ./start_web.sh
-tmux new-window -t rematch -d ./start_celery.sh
-tmux new-window -t rematch
+if [ -n "$IN_DOCKER" ]; then
```

Comment:
I think this is no longer needed, server is started as a service. I think we should revert this.